### ISSUE TITLE
Pretty printing swallows empty namespaces

### ISSUE THREAD

**Issue by joewiz** [eXist-db/eXide#108]
Given the following query:

```
<foo xmlns="a">
  <bar xmlns=""/>
</foo>
```

eXide returns the following result:

```
<foo xmlns="a">
  <bar/>
</foo>
```

Note that the empty namespace declaration on the `<bar>` element is suppressed.

Expected result:

```
<foo xmlns="a">
  <bar xmlns=""/>
</foo>
```

**Comment by joewiz** [comment]
Closed by #168.

**Comment by joewiz** [comment]
Closed by #168.
